src/test/java/io/openliberty/tools/intellij/it/MavenSingleModAppTest.java

```diff
@@ -161,4 +189,104 @@ public void testStartInContainerActionUsingDropDownMenu() {
             }
         }
     }
+
+    /**
+     * Tests dashboard startInContainer/stop actions run from the project's drop-down action menu
+     * and the Liberty tool window toolbar play button.
+     * Notes:
+     * 1, Once issue https://github.com/OpenLiberty/liberty-tools-intellij/issues/299 is resolved,
+     * this method should be moved to SingleModAppTestCommon.
+     * 2. This test is restricted to Linux only because, on other platforms, the docker build process
+     * driven by the Liberty Maven/Gradle plugins take longer than ten minutes. Ten minutes is the
+     * timeout set by the plugins and there is currently no way to extend this timeout through the
+     * Liberty Tools plugin(i.e. set dockerBuildTimeout).
+     */
+    @Test
+    @EnabledOnOs({OS.LINUX})
+    public void testStartInContainerActionUsingPlayToolbarButton() {
+        String testName = "testStartInContainerActionUsingPlayToolbarButton";
+        String absoluteWLPPath = Paths.get(PROJECT_PATH, WLP_INSTALL_PATH).toString();
+
+        // Start dev mode in a container.
+        UIBotTestUtils.runLibertyTWActionFromDropDownView(remoteRobot, "Start in container", true);
+        try {
+            // Validate that the project started.
+            TestUtils.validateProjectStarted(testName, SM_MP_PROJECT_RES_URI, SM_MP_PROJECT_PORT, projExpectedOutput, absoluteWLPPath, true);
+        } finally {
+            if (TestUtils.isServerStopNeeded(absoluteWLPPath)) {
+                // Stop dev mode.
+                UIBotTestUtils.runLibertyTWActionFromDropDownView(remoteRobot, "Stop", true);
+
+                // Validate that the server stopped.
+                TestUtils.validateLibertyServerStopped(testName, absoluteWLPPath);
+            }
+        }
+    }
+
+    /**
+     * Tests dashboard startInContainer/stop actions run from the project's pop-up action menu.
+     * Notes:
+     * 1, Once issue https://github.com/OpenLiberty/liberty-tools-intellij/issues/299 is resolved,
+     * this method should be moved to SingleModAppTestCommon.
+     * 2. This test is restricted to Linux only because, on other platforms, the docker build process
+     * driven by the Liberty Maven/Gradle plugins take longer than ten minutes. Ten minutes is the
+     * timeout set by the plugins and there is currently no way to extend this timeout through the
+     * Liberty Tools plugin(i.e. set dockerBuildTimeout).
+     */
+    @Test
```

Comment:
No not really. I am trying to avoid adding that annotation to all of the tests because of system resource consumption. I only add that annotation to those tests that are more prone to intermittent issues.